sw/device/tests/dif/dif_plic_unittest.cc

```diff
@@ -114,7 +114,7 @@ class IrqEnableSetTest : public IrqTests {
           RV_PLIC_IE01_REG_OFFSET, RV_PLIC_IE01_E63,
       },
       {
-          RV_PLIC_IE02_REG_OFFSET, RV_PLIC_IE02_E82,
+          RV_PLIC_IE02_REG_OFFSET, RV_PLIC_IE02_E81,
```

Comment:
I think you have to do the same for LE and IP. CI currently fails when building this unit test:
```
FAILED: c++ -Isw/device/tests/dif/64cbfc5@@dif_plic_unittest@exe   -I/home/vsts/work/1/s -I/home/vsts/work/1/a/build-out -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Werror -std=c++14 -g -pthread -isystem../../s/sw/vendor/google_googletest/googlemock -isystem../../s/sw/vendor/google_googletest/googlemock/include -isystem../../s/sw/vendor/google_googletest/googletest -isystem../../s/sw/vendor/google_googletest/googletest/include -DMOCK_MMIO -MD -MQ 'sw/device/tests/dif/64cbfc5@@dif_plic_unittest@exe/dif_plic_unittest.cc.o' -MF 'sw/device/tests/dif/64cbfc5@@dif_plic_unittest@exe/dif_plic_unittest.cc.o.d' -o 'sw/device/tests/dif/64cbfc5@@dif_plic_unittest@exe/dif_plic_unittest.cc.o' -c ../../s/sw/device/tests/dif/dif_plic_unittest.cc
../../s/sw/device/tests/dif/dif_plic_unittest.cc:165:35: error: ‘RV_PLIC_LE2_LE82’ was not declared in this scope
           RV_PLIC_LE2_REG_OFFSET, RV_PLIC_LE2_LE82,
                                   ^
../../s/sw/device/tests/dif/dif_plic_unittest.cc:167:3: error: could not convert ‘{{12, 31}, {16, 31}, {20, <expression error>}}’ from ‘<brace-enclosed initializer list>’ to ‘std::vector<dif_plic_unittest::{anonymous}::IrqTests::Register>’
   };
   ^
../../s/sw/device/tests/dif/dif_plic_unittest.cc:266:35: error: ‘RV_PLIC_IP2_P82’ was not declared in this scope
           RV_PLIC_IP2_REG_OFFSET, RV_PLIC_IP2_P82,
                                   ^
../../s/sw/device/tests/dif/dif_plic_unittest.cc:268:3: error: could not convert ‘{{0, 31}, {4, 31}, {8, <expression error>}}’ from ‘<brace-enclosed initializer list>’ to ‘std::vector<dif_plic_unittest::{anonymous}::IrqTests::Register>’
   };
```

Reply:
Ah right, thanks! Looks like I missed those...